src/app/components/About.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const About = () => {
+  return (
+    <section className='flex flex-col items-center gap-4 mb-4'>
+        <h3 className='text-3xl font-bold'>About Us</h3>
+        <article className='font-main text-xl'>The Women Coders Network is a community that fosters connection, ignites knowledge sharing, celebrates the brilliance of women across every facet of tech, and signifies the boundless potential of women shaping the future of technology, together!</article>
```

Comment:
If you are going to use the main font, can you do it for the whole site instead of just the sections you work on to maintain consistency?

Reply:
Yes, I believe that's the font we are going with